src/features/addTweetMenuItems.css

```diff
@@ -1,3 +1,7 @@
+.dropdown-menu {
+  max-width: 21rem;
```

Comment:
Where does `21rem` come from? Sounds a bit magical 😅

Reply:
For now this was the width that looked best for the default column size.
However, looking at it now it may be worth setting this dynamically if custom column widths are enabled & base the `max-width` on that instead